docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+### How likely is it that you would recommend MUI to a friend or colleague?
+
+As for the Net Promoter Score, this year we also saw a decrease. Last year, we had 62,2% of the respondents as Promoters whereas now we had 56,4%. NPS itself moved from 62 in 2020 to 46 in 2021. If we consider the scale, we're still at the great level, though (30 to 70 is considered great), which makes us glad. We definitely have much more to do to achieve an excellent score. The reasoning for the previous and next question might fit to explain NPS data.
+
+<img src="/static/blog/2021-survey/2.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Detractors: 10.88%, Neutrals: 32.69%, Promoters: 56.43%, Overall NPS: 46. Scale: -100 to 0 needs improvements, 0 to 30 is good, 30 to 70 is great, and 70 to 100 is excellent." />
+
+<p class="blog-description">1563 out of 1589 answered</p>
+
+### What is the main benefit you get from using MUI?
+
+<img src="/static/blog/2021-survey/3.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="This question results in 2019 - Design: 248; Time: 193; DX: 128; Components: 89; Customizability: 53; Documentation: 25; Accessibility: 5; Typescript: 5; Community support: 4; This question results in 2021 - Time: 597; Design: 407; Components: 309; DX: 306; Customizability: 181; Documentation: 57; Community support: 26; Accessibility: 24; Performance: 17" />
+
+<p class="blog-description">1422 out of 1589 answered this question in 2021</p>
+
+Here's what grew in terms of value proposition:
+
+- **Community (x2.9):** On a similar time frame, community size grew by x2.0. It looks like we're witnessing a network effect: with x2 more users, we got x3 more value from the community to the user base.
+- **Accessibility (x2.2):** Developers seem to be noticing the work we have put in, special thanks to Sebastian Silbermann [(@eps1lon)](https://github.com/eps1lon) and his continued help!
+- **Customizability (x1.6):** Apparently, you seem to be appreciating the new capabilities introduced in v5. However, it might still be far behind what we were aiming for, especially in light of the number of requests asking to better solve this problem.
+- **Components (x1.6):** The same goes for components, you seem to be enjoying the newly added components.
+  - Component quality (x1.0): It's encouraging to know you don't seem to be experience a decrease in quality. It's a risk we're taking with growing the MUI team, since it takes time for new team members to master the craft since we set the bar very high.
```

Comment:
```suggestion
  - Component quality (x1.0): You know that we set the bar very high for the quality of our components. That can be challenge as we continue to grow, because we expect nothing but the best from our team. Thankfully, this result shows that our new members have done an excellent job maintaining our standards.
```